ui/src/app/pages/app/component.ts

```diff
@@ -0,0 +1,310 @@
+import {Component, NgZone, OnDestroy, OnInit} from '@angular/core';
```

Comment:
This file was removed, you'll need to move your logic to somewhere around here: https://github.com/all-of-us/workbench/blob/master/ui/src/app/routing/app-routing.tsx#L272